Apps/WebClient/src/ClientApp/app/components/timeline/commentSection.vue

```diff
@@ -174,5 +121,17 @@ export default class CommentSectionComponent extends Vue {
         this.isLoadingComments = false;
       });
   }
+
+  private onCreate(comment: UserComment) {
```

Comment:
they all seem to be doing the same, we can just set a **needsUpdate** from the child component